src/librustdoc/html/static/main.js

```diff
@@ -1547,7 +1547,7 @@ function defocusSearchBar() {
                               item.displayPath + "<span class=\"" + type + "\">" +
                               name + "</span></a></td><td>" +
                               "<a href=\"" + item.href + "\">" +
-                              "<span class=\"desc\">" + escape(item.desc) +
+                              "<span class=\"desc\">" + item.desc +
```

Comment:
This is no longer escaped because it's HTML, right? Can you add a comment saying that?
```suggestion
                              // the description can be HTML, so don't escape it
                              "<span class=\"desc\">" + item.desc +
```
I'm also a little concerned this a change in behavior ... where is the description coming from? What would have been escaped before and isn't now - can you add a test?

Reply:
Where would I add the test: would it be a unit test or an integration test?

Reply:
This is in JavaScript, so it would go in `src/test/rustdoc-js` (the javascript doesn't have unit tests at all). It should test the change in behavior, which I think is that search results will no longer be escaped? I don't know exactly what that would look like, that's one of the reasons I'd like a test.

Reply:
You can check the results returned by the search function, so including the `doc` field that has been updated here to confirm that it still contains HTML tags (and didn't escape them).

Reply:
Done! Finally.